modules/dreamview/backend/simulation_world/simulation_world_service.cc

```diff
@@ -853,10 +860,8 @@ void SimulationWorldService::UpdateSimulationWorld(
 
   for (const Path &path : paths) {
     // Downsample the path points for frontend display.
-    // Angle threshold is about 5.72 degree.
-    constexpr double angle_threshold = 0.1;
-    std::vector<int> sampled_indices =
-        DownsampleByAngle(path.path_points(), angle_threshold);
+    auto sampled_indices =
+        DownsampleByAngle(path.path_points(), kAngleThreshold);
 
     route_paths_.emplace_back();
     RoutePath *route_path = &route_paths_.back();
```

Comment:
route_paths_ is a vector, the pointer address can change.

Reply:
not sure I get this?

Reply:
there will be a problem only when receiving two routing_responese simultaneously, which is unlikely for routing topic.